### ISSUE TITLE
Layer Activations

### ISSUE THREAD

**Issue by Benny-Nottonson** [Benny-Nottonson/voodoo#5]
# Layer Activations

[Layer activations](https://keras.io/api/layers/activations/)
[relu function](https://keras.io/api/layers/activations/#relu-function)
[sigmoid function](https://keras.io/api/layers/activations/#sigmoid-function)
[softmax function](https://keras.io/api/layers/activations/#softmax-function)
[softplus function](https://keras.io/api/layers/activations/#softplus-function)
[softsign function](https://keras.io/api/layers/activations/#softsign-function)
[tanh function](https://keras.io/api/layers/activations/#tanh-function)
[selu function](https://keras.io/api/layers/activations/#selu-function)
[elu function](https://keras.io/api/layers/activations/#elu-function)
[exponential function](https://keras.io/api/layers/activations/#exponential-function)
[leaky_relu function](https://keras.io/api/layers/activations/#leaky_relu-function)
[relu6 function](https://keras.io/api/layers/activations/#relu6-function)
[silu function](https://keras.io/api/layers/activations/#silu-function)
[gelu function](https://keras.io/api/layers/activations/#gelu-function)
[hard_sigmoid function](https://keras.io/api/layers/activations/#hard_sigmoid-function)
[linear function](https://keras.io/api/layers/activations/#linear-function)
[mish function](https://keras.io/api/layers/activations/#mish-function)
[log_softmax function](https://keras.io/api/layers/activations/#log_softmax-function)
